config/etc/system-image/writable-paths

```diff
@@ -13,27 +13,18 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
-/var/crash                              auto                    persistent  none        none
-/var/lib/AccountsService/users          auto                    persistent  none        none
 # snap data
 /var/snap                              auto                    persistent  transition  none
-/var/lib/bluetooth                      auto                    persistent  none        none
 # cloud-init
 /etc/cloud                              auto                    persistent  none        none
 /var/lib/cloud                          auto                    persistent  none        none
-/var/lib/colord                         auto                    persistent  transition  none
+# dbus machine-id
 /var/lib/dbus                           auto                    persistent  none        none
+# FIXME: do we need this in the world of networkd?
 /var/lib/dhcp                           auto                    persistent  none        none
```

Comment:
we still get a dhclient up before console-conf runs. do we want to drop this completely (i was never sure if thats a bug or a feature) ?